templates/contribute/style.md

```diff
@@ -503,6 +503,9 @@ example {x y : ℝ} (hxy : x ≤ y) (h : ∀ ε > 0, y - ε ≤ x) : x = y :=
     linarith  
 ```
 
+Both when using the rewrite tactic (`rw`) and the `do` notation, there should
+be a space after the left arrow (`←`).
```

Comment:
Can you give an example of each?

Reply:
Let's not mention the do notation, since this moslty relevant in tactics.
I added an example, not sure if it is a good one...